### ISSUE TITLE
Hint label requires a default

### ISSUE THREAD

**Issue by ChrisKnightLDN** [Greater-London-Authority/ldn-viz-tools#431]
The hint label on inputs is optional - but there should be a default label.
![Screenshot 2024-07-25 at 17 31 54](https://github.com/user-attachments/assets/7b65aebb-eea7-446b-8d52-fe47404a7e2c)

This is for accessibility and ux. NOT displaying a label should be an exception and may only be applicable in certain sidebar control situations. Even in these situations it would probably be preferable to provide a group title/ instruction than hide behind a icon-only hint label.
(figma):
![Frame 3222](https://github.com/user-attachments/assets/df3c1b0f-49ab-4f7f-8304-fd7a6e3fe483)

need to look more closely at the use cases where this icon-only approach has been applied.

**Comment by ChrisKnightLDN** [comment]
also see #430 about correct icon use
